app/src/conferences/models.py

```diff
@@ -112,11 +112,44 @@ class Meta:
     price_transport = models.IntegerField(
         verbose_name=_('Price for transportation')
     )
+    price_transport_with_discount = models.IntegerField(
+        verbose_name=_('Price for transportation with discount'),
+        default=0
+    )
+    price_transport_baggage = models.IntegerField(
+        verbose_name=_('Price for transport baggage'),
+        default=0
+    )
     price_base = models.IntegerField(
         verbose_name=_('Organisation fee'),
         default=0
     )
 
+    first_discount = models.IntegerField(
```

Comment:
Maybe you could create a separate `Discounts` model for these discounts with `OneToOneField` to `Zosia` model?